index.html

```diff
@@ -13,7 +13,7 @@
         <img src="icons/Icon - Menu.svg" alt="" />
       </nav>
       <section class="container-intro">
-        <h1>Hey there.<br />I’m Victor.<br /></h1>
+        <h1>Hey there.<br/>I’m Victor.<br/></h1>
         <p>
```

Comment:
https://github.com/victormc13/Portfolio-project/blob/6fdaa8248a8121ad63c10cdbf3154bf7236a96d4/index.html#L13

- [ ]  Please, ensure that your `img` tag always has an `alt` attribute this is good for accessibility reasons, for example, for a visually impaired individual or a slow internet user. 😃